editor/export/project_export.cpp

```diff
@@ -1794,16 +1791,23 @@ ProjectExportDialog::ProjectExportDialog() {
 	sections->connect("tab_changed", callable_mp(this, &ProjectExportDialog::_tab_changed));
 
 	// Disable by default.
-	name->set_editable(false);
-	export_path->hide();
-	advanced_options->set_disabled(true);
-	runnable->set_disabled(true);
 	duplicate_preset->set_disabled(true);
 	delete_preset->set_disabled(true);
 	script_key_error->hide();
-	sections->hide();
 	parameters->edit(nullptr);
 
+	// Label shown when no presets are present.
+
+	empty_label = memnew(Label(TTR("No presets found.\nCreate one so that its parameters can be edited here.")));
+	empty_label->set_horizontal_alignment(HORIZONTAL_ALIGNMENT_CENTER);
+	empty_label->set_vertical_alignment(VERTICAL_ALIGNMENT_CENTER);
+	empty_label->set_autowrap_mode(TextServer::AUTOWRAP_WORD_SMART);
+	empty_label->set_clip_text(true); // Necessary to avoid overexpanding the dialog vertically.
```

Comment:
On minimum size, the list is completely invisible. Why would you want that?

<img width="614" height="136" alt="Image" src="https://github.com/user-attachments/assets/d46bdaa6-212f-4283-a573-cfccfd12bab1" />

With the extra line:

<img width="614" height="161" alt="Image" src="https://github.com/user-attachments/assets/3251af2b-2cb4-431a-809d-54b2e3ebb2a4" />